provider/aws.go

```diff
@@ -189,6 +189,7 @@ func (p *AWSProvider) ApplyChanges(_ string, changes *plan.Changes) error {
 func (p *AWSProvider) submitChanges(changes []*route53.Change) error {
 	// return early if there is nothing to change
 	if len(changes) == 0 {
+		log.Infoln("All records are already up to date")
```

Comment:
I think the `log` packages typically add the line break, so `log.Info` should be fine.